### ISSUE TITLE
Core dump when is not null predicate

### ISSUE THREAD

**Issue by chaoyli** [apache/incubator-doris#2218]
After add a column after schema change.
A DefaultValueReader will be created for this column.
Upon IS NOT NULL predicate, be will be core dump.

**Comment by chaoyli** [comment]
You can reproduce this bug with the following procedure.
1. create table 
```
CREATE TABLE `test` (
  `k1` tinyint(4) NOT NULL COMMENT "",
  `k2` smallint(6) NOT NULL COMMENT "",
  `k3` int(11) NOT NULL COMMENT "",
  `k4` bigint(20) NOT NULL COMMENT "",
  `k5` decimal(9, 3) NOT NULL COMMENT "",
  `k6` char(5) NOT NULL COMMENT "",
  `k10` date NOT NULL COMMENT "",
  `k11` datetime NOT NULL COMMENT "",
  `k7` varchar(20) NOT NULL COMMENT "",
  `k8` double MAX NOT NULL COMMENT "",
  `k9` float SUM NOT NULL COMMENT ""
) ENGINE=OLAP
AGGREGATE KEY(`k1`, `k2`, `k3`, `k4`, `k5`, `k6`, `k12`, `k10`, `k11`, `k7`)
COMMENT "OLAP"
DISTRIBUTED BY HASH(`k1`) BUCKETS 1
PROPERTIES (
"storage_type" = "COLUMN"
);
```

2. add a column
```
alter table test add column k12 int not null default '0' after k6
```

3. send a query
```
select * from test where k12 is not null
```

**Comment by chaoyli** [comment]
The bug is fixed in #2219
